akka-docs/src/main/paradox/stream/operators/Sink/foreach.md

```diff
@@ -4,23 +4,34 @@ Invoke a given procedure for each element received.
 
 @ref[Sink operators](../index.md#sink-operators)
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [Sink.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala) { #forEach }
-
-@@@
+@apidoc[Sink.foreach](Sink$) { java="#foreach(akka.japi.function.Procedure)" scala="#foreach[T](f:T=%3EUnit):akka.stream.scaladsl.Sink[T,scala.concurrent.Future[akka.Done]]" }
 
 ## Description
 
 Invoke a given procedure for each element received. Note that it is not safe to mutate shared state from the procedure.
 
-The sink materializes into a  @scala[`Future[Option[Done]]`] @java[`CompletionStage<Optional<Done>`] which completes when the
+The sink materializes into a @scala[`Future[Done]`] @java[`CompletionStage<Done>`] which completes when the
 stream completes, or fails if the stream fails.
 
 Note that it is not safe to mutate state from the procedure.
 
+See also:
+
+* @ref[`foreachAsync`](foreachAsync.md) Invoke a given procedure asynchronously for each element received.
+* @ref[`actorRef`](actorRef.md) Send the elements from the stream to an `ActorRef`.
```

Comment:
Curious why you feel this `Sink` is relevant?

Reply:
To point people to `Sink.actorRef` instead of having `actor ! _` in `Sink.foreach`.